rules/windows/registry/registry_event/registry_event_susp_process_registry_modification.yml

```
@@ -0,0 +1,36 @@
+title: Registry Modification by Suspicious Processes
```

Just because you used a registry event category

Also please change the filename to reflect this suggestion too.

```suggestion
title: Registry Tampering by Potential Suspicious Processes
```